sql/parser/Makefile

```diff
@@ -0,0 +1,3 @@
+sql.go: sql.y
+	go tool yacc -o sql.go sql.y
+	@echo "$$(echo '// Code generated by go yacc.' | cat - sql.go)" > sql.go
```

Comment:
Yeah, the buffering is necessary; i was optimistically hoping that it would work if your `echo` one did. `sed -i` would be convenient here but all the ways of dealing with newlines in sed are awful and I'm not sure about getting the escaping right through a `Makefile`.

Can `go tool yacc` write to stdout?